migration_original/ODS1Stage/tables/Base/ProviderToClientProductToDisplayPartner/spu_translated_ProviderToClientProductToDisplayPartner.sql

```diff
@@ -83,13 +78,11 @@ insert_statement := ' insert (
 ---------------------------------------------------------  
 
 merge_statement := 'merge into base.providertoclientproducttodisplaypartner as target
-using
-('||select_statement||') as source
-on source.providerid = target.providerid
-and source.clienttoproductid = target.clienttoproductid
-and source.syndicationpartnerid = target.syndicationpartnerid
-WHEN MATCHED then delete
-when not matched then' || insert_statement;
+                    using ('||select_statement||') as source
+                    on source.providerid = target.providerid
+                        and source.clienttoproductid = target.clienttoproductid
+                        and source.syndicationpartnerid = target.syndicationpartnerid
```

Comment:
add when matched then delete

Reply:
Not needed anymore after our discussion today